recipes/_nvidia_install.rb

```diff
@@ -28,14 +28,19 @@
     not_if { ::File.exist?('/usr/bin/nvidia-smi') }
   end
 
+
+  nvidia_install_command = "./nvidia.run --silent --dkms"
+  if node['platform'] == 'ubuntu' && node['platform_version'] == "18.04"
```

Comment:
I would add a comment here explaining why this is needed at the moment, just to keep track of why we did this patch.

Otherwise LGTM

Reply:
done